### DISCUSSION TITLE
Face issue connection does not support MultipleActiveResultSets.

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11436] [Q&A]
When application initially load it say:

![image](https://user-images.githubusercontent.com/5202291/160210890-34b2ee47-062b-4e88-94f2-31d5da4ebe08.png)

When refresh multiple times or restart application so it gone, please help that case give bad impression.

**Comment by hishamco** [comment]
Could you try to add `MultipleActiveResultSets=True` to your ConnectionString?

**Comment by saleemhaq** [comment]
Please help me out in that concern:
https://github.com/OrchardCMS/OrchardCore/discussions/11429

**Reply by hishamco** [reply]
Is the above answer solves your issue?

**Reply by saleemhaq** [reply]
> Is the above answer solves your issue?

Now after put this @hishamco i face this issue below:

InvalidOperationException: Operations that change non-concurrent collections must have exclusive access. A concurrent update was performed on this collection and corrupted its state. The collection's state is no longer correct.

![image](https://user-images.githubusercontent.com/5202291/161141415-81d749ce-b390-4a15-b87b-6466db1358b0.png)

**Comment by jtkech** [comment]
Most of the time, when we have a MARS issue it is because a non awaited async call, e.g. in custom code.

But it may depend on the OC version you are using.

**Reply by jtkech** [reply]
For example see https://github.com/OrchardCMS/OrchardCore/issues/7449#issuecomment-721049949

**Reply by saleemhaq** [reply]
> Most of the time, when we have a MARS issue it is because a non awaited async call, e.g. in custom code.
> 
> But it may depend on the OC version you are using.

@jtkech please help on issue below:

InvalidOperationException: Operations that change non-concurrent collections must have exclusive access. A concurrent update was performed on this collection and corrupted its state. The collection's state is no longer correct.

![image](https://user-images.githubusercontent.com/5202291/161141415-81d749ce-b390-4a15-b87b-6466db1358b0.png)

**Comment by jtkech** [comment]
> Now after put this @hishamco i face this issue below:

Which seems to confirm a non awaited task, maybe a missing await when calling an async method in any custom code.

If you have any custom code can you share it, ideally to be able to repro ;)
